TurboMinerVirtualTable.Unity/Assets/Scripts/Ui/Lobby/PlayerManager.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System.Linq;
+using TMPro;
+
+public class PlayerManager : MonoBehaviour
```

Comment:
nazwa ździebko chujowa, brzmi jakby to byla klasa do zarzadzania graczem ogolnie, nawet w trakcie gry, a to tylko uklada labelki.
PlayerLabelsMaganer bettah, ale moze wymyslisz cos lepszego

Reply:
no nad tą nazwą w sumie kminiłem i dałem taką roboczo, a potem zapomniałem, żeby zmienić, coś pokminie